var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -73,6 +75,11 @@ class Cp2k(Package):
 
     parallel = False
 
+    def setup_environment(self, spack_env, run_env):
+        # The trailing slash in the data_path has to be there.
+        data_path = os.path.join(self.prefix, 'data/')
```

Comment:
I think it should be `os.path.join(pre, 'data{0}'.format(os.path.sep))`

Reply:
And now, using the brand new Github suggestion feature:
```suggestion
        data_path = os.path.join(self.prefix, 'data{0}'.format(os.path.sep))
```